docs/snippets/protocol-sdk/create/createNew1155Token.ts

```diff
@@ -8,10 +8,7 @@ const { parameters } = await creatorClient.create1155({
   // by providing a contract address, the token will be created on an existing contract
   // at that address
   contract: collectionAddress, // [!code hl]
-  token: {
-    // token metadata uri
-    tokenMetadataURI: "ipfs://DUMMY/token.json",
-  },
+  tokenMetadataURI: "ipfs://DUMMY/token.json",
```

Comment:
thank you - our build system should have caught this

Reply:
I actually believe the documentation is correct (this was moved to be within `token`):
https://github.com/ourzora/zora-protocol/blob/main/packages/protocol-sdk/src/create/types.ts#L32

do you have the latest version of `@zoralabs/protocol-sdk` ?